fhd_output/fhd_quickview.pro

```diff
@@ -276,12 +277,13 @@ ENDFOR
 IF Keyword_Set(cal_source_flag) THEN cal_sources = source_image_generate(cal_source_arr_out,obs_out,pol_i=4,resolution=16,$
     dimension=dimension,restored_beam_width=restored_beam_width,_Extra=extra)
 
-; renormalize based on weights
 renorm_factor = get_image_renormalization(obs_out,weights_arr=weights_arr,beam_base=beam_base_out,filter_arr=filter_arr,$
   image_filter_fn=image_filter_fn,pad_uv_image=pad_uv_image,degpix=degpix,/antialias)
 for pol_i=0,n_pol-1 do begin
   *instr_dirty_arr[pol_i]*=renorm_factor[pol_i]
   IF model_flag THEN *instr_model_arr[pol_i]*=renorm_factor[pol_i]
+  IF source_flag THEN *instr_sources[pol_i]*=renorm_factor[pol_i]
+  IF Keyword_Set(ring_radius) THEN *instr_rings[pol_i]*=renorm_factor[pol_i]
```

Comment:
`renorm_factor` is now either 1 for `filter_uv_weighted` or set to the average calculated from all polarizations. Doesn't that mean it never actually is different for the different polarizations? I think we need to decide whether it *should* always be the same for all polarizations and make `renorm_factor` a float (instead of an array), or remove the bit at the end of `get_image_renormalization` that sets each element equal to the mean value.

That investigation could be done as a new branch and PR, though, if you'd like to get these changes merged first.

Reply:
I do think that is should be per-polarization for all image filters. PR #182 does that (or did that, it needs to be updated in light of recent changes). I think it would make sense to do that as a separate PR but we can also implement it here.

Reply:
If that is coming soon, just leave this code as it is. That will make it easier to isolate the changes if something looks broken later.